tests.py

```diff
@@ -16,8 +16,12 @@ def redis():
     redis.delete(TEST_KEY)
 
 
-def get_node(redis, n_replicas, total_replicas):
-    node = RingNode(redis, TEST_KEY, n_replicas=n_replicas)
+def get_node(
+    redis, n_replicas, total_replicas, hash_algorithm=HashAlgorithm.CRC32
```

Comment:
Just make `hash_algorithm` a required arg to avoid missing it?

Reply:
That's not backwards-compatible, but I guess we don't care? I think that's better. I'll make it required and bump the version to `0.6.0` when I release to indicate a breaking change to any external users.